app/pages/stacking/components/stacking-terms.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Box, Flex, Text } from '@blockstack/ui';
+
+export const StackingTerms = () => {
+  return (
+    <Box textStyle={['caption', 'body.large']} mt="loose">
+      <Flex alignItems="center">
+        <Box width={['12px', '16px']} mr="base-tight">
+          <svg
+            width="15"
+            height="16"
+            viewBox="0 0 12 16"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
```

Comment:
these icons could be their own components

Reply:
Done